sydent/http/servlets/store_invite_servlet.py

```diff
@@ -133,6 +133,9 @@ def render_POST(self, request: Request) -> JsonDict:
 
         substitutions["ephemeral_private_key"] = ephemeralPrivateKeyBase64
         if substitutions["room_name"] != "":
+            if len(substitutions["room_name"]) > 30:
+                substitutions["room_name"] = substitutions["room_name"][:25] + "…"
```

Comment:
Curious why 25 instead of 29. Is that because of the `"(%s) "` below adding 3 characters? In which case we can put 26 here?

Otherwise just a nit and doesn't really matter all that much.

Reply:
Eh, just arbitrary really, as it always irrationally annoys me when things truncate stuff when there's only a couple of extra characters :shrug: